src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -129,8 +129,11 @@ export class BrowsingContextProcessor {
       // debugger  and ignore them.
       targetCdpClient
         .sendCommand('Runtime.runIfWaitingForDebugger')
+        .catch((error) => this.#logger?.(LogType.system, error))
```

Comment:
ESLint suggests to move this after `then` with `'@typescript-eslint/no-floating-promises'`. Does it make sense? If yes, please do so.